bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gmos/GmosNorthType.java

```diff
@@ -80,27 +80,29 @@ public StageModeNorth parse(String name, StageModeNorth defaultValue) {
         }
     };
 
-    public static enum DisperserNorth implements GmosCommonType.Disperser, ObsoletableSpType {
-        MIRROR("Mirror", "mirror"),
-        B1200_G5301("B1200_G5301", "B1200"),
-        R831_G5302("R831_G5302", "R831"),
-        B600_G5303("B600_G5303", "B600") {public boolean isObsolete() {return true;}},
-        B600_G5307("B600_G5307", "B600"),
-        R600_G5304("R600_G5304", "R600"),
-        R400_G5305("R400_G5305", "R400"),
-        R150_G5306("R150_G5306", "R150"),
-        R150_G5308("R150_G5308", "R150"),
+    public enum DisperserNorth implements GmosCommonType.Disperser, ObsoletableSpType {
+        MIRROR("Mirror", "mirror", 0),
+        B1200_G5301("B1200_G5301", "B1200", 1200),
+        R831_G5302("R831_G5302", "R831", 831),
+        B600_G5303("B600_G5303", "B600", 600) {public boolean isObsolete() {return true;}},
+        B600_G5307("B600_G5307", "B600", 600),
+        R600_G5304("R600_G5304", "R600", 600),
+        R400_G5305("R400_G5305", "R400", 400),
+        R150_G5306("R150_G5306", "R150", 150),
+        R150_G5308("R150_G5308", "R150", 150),
         ;
 
         /** The default Disperser value **/
         public static DisperserNorth DEFAULT = MIRROR;
 
-        private String _displayValue;
-        private String _logValue;
+        private final String _displayValue;
+        private final String _logValue;
+        private final int _linesPerMm;
```

Comment:
I think it would be best to name this what it actually represents, not its units. Looking at http://www.gemini.edu/sciops/instruments/gmos/spectroscopy-overview/gratings, looks like we are trying to model the Gmos gratings' "Ruling Density", which happens to be measured in lines per mm. So, my initial thought is that this should be called "rulingDensity" or something like that. As it is, it's like calling "metersPerSecond" something that is actually representing velocity. What do you think?